amoro-ams/pom.xml

```diff
@@ -503,6 +564,70 @@
                     <skip>true</skip>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>org.openapitools</groupId>
+                <artifactId>openapi-generator-maven-plugin</artifactId>
```

Comment:
the latest version is [7.9.0](https://mvnrepository.com/artifact/org.openapitools/openapi-generator-maven-plugin/7.9.0) why we don't use it?